trex-container-app/app/Dockerfile

```
@@ -15,6 +22,12 @@ ENV PYTHONPATH="/opt/trex/trex-core/scripts/automation/trex_control_plane/intera
 ENV TREX_DIR="/opt/trex/trex-core/scripts"
 ENV TRAFFICGEN_DIR="/opt/trafficgen"
 
+# Copy scripts
 COPY scripts /usr/local/bin/
+COPY --from=build /scripts/webserver /usr/local/bin/
+
+# Update permissions
+RUN chmod +x /usr/local/bin/webserver
```

Avoid copying anything that is not needed in the image that is inside the scripts, e.g. the webserver.go

```suggestion
COPY --chmod=550 --from=build /scripts/webserver /usr/local/bin/
```